Sources/ShortcutFoundation/Extensions/Int+Extension.swift

```diff
@@ -0,0 +1,22 @@
+//
+//  Int+Extension.swift
+//  ShortcutFoundation
+//
+//  Created by Andreas Lif on 2022-01-12.
+//  Copyright © 2021 Shortcut Scandinavia Apps AB. All rights reserved.
+//
+
+import Foundation
+
+public extension Int {
+    
+    var isEven: Bool {
+        guard self != 0 else { return true }
+        return self % 2 == 0
+    }
+    var isOdd: Bool {
+        guard self != 0 else { return false }
```

Comment:
@AndreasShortcut could we try:

return !self.isEven

Reply:
I did `!(self.isMultiple(of: 2))` instead in 5fcd885. What do you think?

Reply:
Changed again, to your suggestion, in 5fcbeba